test/integration/targets/pip/tasks/pip.yml

```diff
@@ -122,6 +122,38 @@
     that:
       - "not url_installed.changed"
 
+# Test user and constraints installs
+- name: create a constraints file
+  copy: dest={{ output_dir }}/pipconstraints.txt
+    content="iso8601===0.1.11"
+
+- name: install iso8601 with constraints applied
+  pip:
+    name: iso8601
+    constraints: "{{ output_dir }}/pipconstraints.txt"
+    user: yes
+
+# TODO pip module does not report back version in the return info.
+# Once it does/can, change this to verify using req_installed.version instead
+# of looking in stdout_lines
+- name: check that version of package was installed that we expect
+  assert:
```

Comment:
This check should be after the register task.

Reply:
Heh. Duh. Thanks.